pkg/sources/fake/fake.go

```diff
@@ -6,15 +6,36 @@ import (
 
 	"telemetry/pkg/msgs"
 	"telemetry/pkg/pubsub"
+	"telemetry/pkg/util/randutil"
 )
 
 const (
 	// interval between fake data readings
-	readInterval = 20 * time.Millisecond
+	readInterval = 200 * time.Millisecond
+
+	// TODO(karlding): change these to the actual values
+	rightTurnOn     = 1
```

Comment:
Yeah, that would be ideal. Not sure how messages are going to work, since a lot of that seems to be up in the air.